translate/cloud-client/hybrid_glossaries/hybrid_tutorial.py

```diff
@@ -0,0 +1,227 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START translate_hybrid_imports]
+# Imports the Google Cloud client libraries
+from google.cloud import translate_v3beta1 as translate
+from google.cloud import vision
+from google.cloud import texttospeech
+
+import io
+import os
+# [END translate_hybrid_imports]
+
+# [START translate_hybrid_project_id]
+# extract GCP project id
+PROJECT_ID = os.environ['GCLOUD_PROJECT']
+# [END translate_hybrid_project_id]
+
+
+# [START translate_hybrid_vision]
+def pic_to_text(infile):
+    """Detects text in an image file
+
+    ARGS
+    infile: path to image file
+
+    RETURNS
+    String of text detected in image
+    """
+
+    # Instantiates a client
+    client = vision.ImageAnnotatorClient()
+
+    # Opens the input image file
+    with io.open(infile, 'rb') as image_file:
+        content = image_file.read()
+
+    image = vision.types.Image(content=content)
+
+    # For dense text, use document_text_detection
+    # For less dense text, use text_detection
+    response = client.document_text_detection(image=image)
+    text = response.full_text_annotation.text
+
+    return text
+    # [END translate_hybrid_vision]
+
+
+# [START translate_hybrid_create_glossary]
+def create_glossary(languages, project_id, glossary_name, glossary_uri):
+    """Creates a GCP glossary resource
+    Assumes you've already manually uploaded a glossary to Cloud Storage
+
+    ARGS
+    languages: list of languages in the glossary
+    project_id: GCP project id
+    glossary_name: name you want to give this glossary resource
+    glossary_uri: the uri of the glossary you uploaded to Cloud Storage
+
+    RETURNS
+    nothing if glossary creation is successful
+    """
+
+    # Instantiates a client
+    client = translate.TranslationServiceClient()
+
+    # Designates the data center location that you want to use
+    location = 'us-central1'
+
+    # Set glossary resource name
+    name = client.glossary_path(
+        project_id,
+        location,
+        glossary_name)
+
+    # Set language codes
+    language_codes_set = translate.types.Glossary.LanguageCodesSet(
+        language_codes=languages)
+
+    gcs_source = translate.types.GcsSource(
+        input_uri=glossary_uri)
+
+    input_config = translate.types.GlossaryInputConfig(
+        gcs_source=gcs_source)
+
+    # Set glossary resource information
+    glossary = translate.types.Glossary(
+        name=name,
+        language_codes_set=language_codes_set,
+        input_config=input_config)
+
+    resource = client.location_path(project_id, location)
+
+    # Create glossary resource
+    operation = client.create_glossary(parent=resource, glossary=glossary)
+
+    return operation.result(timeout=90)
```

Comment:
If you need to wait for the result of the operation. You should make sure to clarify that with a comment. 

Probably not worth doing a return though, if the result is None